Makefile

```diff
@@ -57,7 +57,7 @@ DD_ENV=production
 # make sure we only ever go to production with `--tls=enable`
 USE_TLS=enable
 CARGO_PROFILE=release
-RUST_LOG?=shuttle=debug,info
+RUST_LOG?=nbuild-core=warn,shuttle=debug,info
```

Comment:
Could we achieve this by passing a layer with an envfilter to the `setup_tracing` in builder?

Reply:
Done here: [2edaf12](https://github.com/shuttle-hq/shuttle/pull/1253/commits/2edaf12c10897b00a60881730c3e9259466df522)